### DISCUSSION TITLE
Improving site UI

### DISCUSSION THREAD

**Discussion by manimonji** [robocode-dev/tank-royale#18] [Ideas]
<h3>I'm sure you know that the Robocode Tank Royal Docs UI <strong>"IS NOT VERY GOOD".</strong></h3>
<h2>But Why?</h2>

-  Dark colors on dark background (Dark green text on a darker background)
-  No hover animations.
-  Header must be darker than body

<img src='https://user-images.githubusercontent.com/69511102/164888297-43023e2c-5e89-42f9-9f0b-31c7d2762ef5.png'>
<h3>But we can improve it like this:</h3>
<img src='https://user-images.githubusercontent.com/69511102/164912888-201c2a17-1fa0-4fe1-a216-b384843ea0ca.png'>
 But we can not use this as an excuse to complain about @flemming-n-larsen . Instead, we can try to improve the site UI and tell to @flemming-n-larsen to fix and replace codes. I'm a web developer (I don't say I am a grand master) and I need your feedback. I know this site is not a grocery store, but the first thing a user looks at is the appearance of the site. I hope you do not think that my intention is to insult you and @flemming-n-larsen .
Understand that I do not mean that the site is very bad. The website needs some little fixes.
<h1>Click On Images For Batter Quality</h1>

**Comment by flemming-n-larsen** [comment]
Thank you for the feedback @manimonji. 🙂
My focus on mainly on the game itself, but you are right that the web page needs attention as well. Hense, I welcome help with improving the documentation and site.

So you are welcome to give it a shot doing a PR. Perhaps a small one to begin with to understand what you are trying to achive first.
Perhaps it is the styling first of all, you want to play with. That should be easy. It is located under [buildDocs/docs/.vuepress/styles/index.styl](https://github.com/robocode-dev/tank-royale/blob/master/buildDocs/docs/.vuepress/styles/index.styl)

The [Robocode Tank Royale Docs](https://robocode-dev.github.io/tank-royale/) is made with [VuePress](https://vuepress.vuejs.org/) using plain markup and plugins, which makes it easy to write documentation the same way as the ReadMe.md files. I want to stick with the markdown syntax and keep it simple to maintain the documentation.

And it is important to incorporate the various documentations from e.g. [Javadoc](https://www.baeldung.com/javadoc) and [Docfx](https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html), i.e. generated static HTML. And in the future more documentation formats might arrive for API for other platforms besides Java/JVM and .Net. But this should be trivial as long as the generated API documentation is kept under their own directories on the GitHub docs.

**Reply by manimonji** [reply]
Thank You Man!

**Reply by flemming-n-larsen** [reply]
Hi @manimonji ,

I have now updated the online documentation according to the font, styles, and colors like yo proposed.
You can see the result here:
https://robocode-dev.github.io/tank-royale/

I will close your [PR #1](https://github.com/robocode-dev/tank-royale/pull/19) as the changes was done directly in the code with VuePress running to see the changes.

**Reply by manimonji** [reply]
I'm happy to hear that!!! i will fix another text color too!

**Reply by flemming-n-larsen** [reply]
I look forward to that. 🙂

And I will be happy to help you out with adjusting the code for changing
the color too:
https://github.com/robocode-dev/tank-royale/blob/master/buildDocs/docs/.vuepress/styles/index.styl
